ChangeLog.d/bugfix_PR3422.txt

```diff
@@ -0,0 +1,2 @@
+Bugfix
+   * Fix building library/net_sockets.c and the ssl_mail_client program on NetBSD. Contributed by Nia Alarie in #3422.
```

Comment:
I'm not going to reject the PR for this because it doesn't break any automation and we'll likely do a pass of reformatting when we prepare the release. But for future reference, please wrap changelog lines to 80 columns.